packages/ceramic-core/package.json

```diff
@@ -37,7 +37,9 @@
     "@ethersproject/base64": "^5.0.2",
     "@ethersproject/providers": "^5.0.0-beta.144",
     "@ethersproject/random": "^5.0.2",
-    "@textile/powergate-client": "^0.2.0",
+    "@pinning-aggregation/aggregation": "^2.1.1",
+    "@pinning-aggregation/common": "^2.0.0",
+    "@pinning-aggregation/ipfs-pinning": "^2.0.1",
```

Comment:
I guess it would be nice for these deps to be part of this monorepo as well?

Reply:
Could be.

Reply:
They are the part of the repo now.

Reply:
Very cool, thanks!